### DISCUSSION TITLE
Solarized Light/Dark UI themes

### DISCUSSION THREAD

**Discussion by rockofox** [community/community#12149] [Other Feature Feedback, Questions, & Ideas]
The solarized themes are pretty iconic, and the light theme is perfect for using light mode while still being easy on the eyes

**Comment by gaknoll** [comment]
Hi @felkr thanks for the feature request! While we don't have a timeline for a potential Solarized theme, it is in our backlog to consider in the future. Right now we are focused on shipping two new colorblind themes.
